### PR TITLE
disable gitauth tests

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees @csrwng FYI / PTAL

### CODE REVIEW

**Comment by gabemontero** [comment]
[testonlyextended]

File: test/extended/builds/gitauth.go [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package builds
 
-import (
-	"fmt"
+// THESE TESTS ARE DISABLED BECAUSE THE XIO.IO DNS HOOK WAS PROVING WAY TOO UNRELIABLE;
```
Comment (csrwng) [inline test/extended/builds/gitauth.go]: why all the shouting :)

s/xio.io/xip.io/

Comment (gabemontero) [inline test/extended/builds/gitauth.go]: shouting ? .... ah, cause of all caps ;-) .... sorry, i'm not that internet savvy 

and @bparees can appreciate my typo there :-)

**Comment by gabemontero** [comment]
[testonlyextended]

**Comment by gabemontero** [comment]
[testonlyextended][extended:core()]

**Comment by bparees** [comment]
lgtm

**Comment by bparees** [comment]
(will wait for the extended run to finish before merging)

**Comment by gabemontero** [comment]
technically, on the extended test run, i just want to make sure the suites get resolved and it does not blow up with an essentially empty .go file present ... I'm going to post if it gets past that point and the tests actually start

**Comment by bparees** [comment]
[test]

**Comment by gabemontero** [comment]
the extended tests launched successfully (402 out of 402 specs in core()) - about 10 tests into it as of this typing

**Comment by bparees** [comment]
[merge]

**Comment by gabemontero** [comment]
conformance failure was a k8s e2e flake

**Pull request merged**